components/directory-server-manager/org.wso2.carbon.directory.server.manager.ui/src/main/resources/web/servicestore/change-passwd-finish.jsp

```diff
@@ -85,14 +86,14 @@
     if (qpplicationComponentFound) {
         if (!isError) {
     %>
-    location.href = '../application/configure-service-provider.jsp?action=update&display=kerberos&spName=<%=spName%>&kerberos=<%=servicePrincipleName%>';
+    location.href = '../application/configure-service-provider.jsp?action=update&display=kerberos&spName=<%=Encode.forJavaScriptBlock(Encode.forUriComponent(spName))%>&kerberos=<%=Encode.forJavaScriptBlock(Encode.forUriComponent(servicePrincipleName))%>';
     <%  } else { %>
-    location.href = '../application/configure-service-provider.jsp?action=cancel&display=kerberos&spName=<%=spName%>';
+    location.href = '../application/configure-service-provider.jsp?action=cancel&display=kerberos&spName=<%=Encode.forJavaScriptBlock(Encode.forUriComponent(spName))%>';
     <%
         }
     }else {
     %>
-    location.href = '<%=forwardTo%>';
+    location.href = '<%=Encode.forJavaScriptBlock(forwardTo)%>';
```

Comment:
Should encode https://github.com/wso2/carbon-identity/pull/989/files#diff-82f092c1cc12afdbf97583a4e3869fdaR75 for Uri